server/services/openai_client.py

```diff
@@ -0,0 +1,43 @@
+from openai import OpenAI
+from dotenv import load_dotenv
+import os
+
+from utilities.prompts.database_schema_representation import *
+
+load_dotenv()
+
+def generate_sql_query_openai(question: str):
+    """
+    Generates an SQL query from a natural language question using the OpenAI API.
+
+    Args:
+        question (str): Natural language question.
+
+    Returns:
+        tuple: A tuple containing the generated SQL query string and the prompt used.
+
+    Raises:
+        RuntimeError: If there is an error with the OpenAI API request.
+    """
+    prompt = (
+        f"### Complete sqlite SQL query only and with no explanation\n"
+        f"### Given the following database schema :\n {code_representation_database_schema}\n"
+        f"### Answer the following: {question}\nSELECT*/"
+    )
+
+    if not question:
+        raise ValueError("Question parameter is required")
```

Comment:
All `error` & `statements` messages should be a constant, it is easier to update them that way